here-positioning-android/gradle.properties

```diff
@@ -0,0 +1,18 @@
+# Project-wide Gradle settings.
```

Comment:
Please align contents with that of other examples (simply copy the file from another example). In particular, you *need* to add `android.enableImprovedDependenciesResolution=true`.

Reply:
Same story as above, this is pretty much copy+paste from other project.

Reply:
Fixed.